src/chainstate/stacks/boot/pox.clar

```diff
@@ -34,7 +34,7 @@
 ;; This function can only be called once, when it boots up
 (define-public (set-burnchain-parameters (first-burn-height uint) (prepare-cycle-length uint) (reward-cycle-length uint) (rejection-fraction uint))
     (begin
-        (asserts! (and is-in-regtest (not (var-get configured))) (err ERR_NOT_ALLOWED))
+        (asserts! (not (var-get configured)) (err ERR_NOT_ALLOWED))
```

Comment:
If we're getting rid of `is-in-regtest`, should we kill it completely?

Reply:
I think we should keep it for the time being -- it may be useful elsewhere.